README.md

```diff
@@ -159,6 +159,10 @@ data directory.
 `tezos-{accuser, endorser}` have configurable node address, so that they can be used with both
 remote and local node.
 
+As an addition, `tezos-baking` package provides `tezos-baking-<network>` services that orchestrate
+systemd units for `tezos-node`, `tezos-baker-<proto>`, and `tezos-endorser-<proto>`.
+Configuration files for these services are located in `/etc/default/tezos-baking-<network>`.
```

Comment:
I think we shouldn't mention this until we create a new native release, just as we don't update the baking doc here either to avoid incorrect informations.